spring-boot-properties/src/test/resources/configuration-processor.properties

```diff
@@ -0,0 +1,2 @@
+com.baeldung.property1 = test1
+com.baeldung.property2 = test2
```

Comment:
I think it would be better if we could use more realistic looking properties with good descriptions. For example, `url` and `timeout`